client/gutenberg/editor/calypsoify-iframe.jsx

```diff
@@ -189,6 +191,23 @@ class CalypsoifyIframe extends Component {
 		}
 	};
 
+	updateImageBlocks = action => {
+		if (
+			! action ||
+			! startsWith( action.data.mime_type, 'image/' ) ||
+			startsWith( action.data.URL, 'blob:' )
+		) {
```

Comment:
maybe add a check for `this.iframePort` just to be consistent with other handlers. I didn't found a case when it fails, but that doesn't mean it can't 😉